.github/workflows/openapi-diff.yml

```
@@ -14,7 +14,7 @@ env:
 
 jobs:
   check-diff:
-    name: build api
+    name: Fails to show the difference between the prod schema and dev schema (optional)
```

Maybe a slightly shorter name so you can read it in the GH actions + put optional at the front?
```suggestion
    name: (Optional) Shows the difference between the prod and dev schema
```